themes/custom/droopler_theme/scss/config/_paths.scss

```diff
@@ -1,3 +1,7 @@
-$theme_path: '/profiles/contrib/droopler/themes/custom/droopler_theme' !default;
+$profile_path: '/profiles/contrib/droopler' !default;
+$theme_path: $profile_path + '/themes/custom/droopler_theme' !default;
 $subtheme_path: '/themes/custom/droopler_subtheme' !default;
 $lato-font-path: '/libraries/lato-webfont/fonts' !default;
```

Comment:
btw to lato tu nam do czegoś potrzebne? w nowej wersji mamy je już w repo theme, wyleciało z libraries